testsuite.zig

```diff
@@ -1,6 +1,6 @@
 const std = @import("std");
 const builtin = @import("builtin");
-const network = @import("network");
+const network = @import("network.zig");
```

Comment:
Is this change really necessary? Apart from that it looks reasonable :)

Reply:
/home/g41797/Devs/ziginstall/zig-linux-x86_64-0.14.0-dev.1694+3b465ebec/zig test --color on /home/g41797/CLionProjects/github.com/g41797/zig-network/testsuite.zig -O Debug
testsuite.zig:3:25: error: no module named 'network' available within module test
const network = @import("**network**");
                        ^~~~~~~~~

Process finished with exit code 1

Reply:
That should definitily work. Is your branch up-to-date?

Reply:
https://github.com/ikskuh/zig-network/blob/10d768bdd5cf2003e4062dce66cde0af906aa2e2/build.zig#L11-L16

Reply:
- zig build test OK
- zig test testsuite.zig - the same error

Reply:
> zig test testsuite.zig - the same error

That is completetly fine and "has to be", while `zig build test` has to work